bundle/manifests/fence-agents-remediation.clusterserviceversion.yaml

```diff
@@ -117,13 +119,22 @@ spec:
         displayName: Shared Parameters
         path: template.spec.sharedparameters
       version: v1alpha1
-  description: "Fence Agents Remediation Operator (FAR)\n\nThe operator will remediate/fence
-    a node when a FenceAgentsRemediation (far) custom resource exists with the node's
-    name.\nFAR uses an existing set of upstream fencing agents for environments with
-    a \ntraditional API end-point (e.g., IPMI) for power cycling cluster nodes.\n\nThe
-    operator is recommended to be used with Node Healthcheck Operator for easier and
-    smoother experience,\nbut it can also work as a standalonde remediator for the
-    more advanced user.\n"
+  description: |
+    Fence Agents Remediation (*FAR*) is a Kubernetes operator that *fence* and remediate unhealthy to healthy nodes.
+    Using a traditional Application Programming Interface (API) FAR runs uses well-known fence agents to remediate a node from an unhealthy state by power cycling the node,
+    and afterwards it delets the node's resources.
+    By doing so, FAR can minimize downtime for stateful applications, restores compute capacity if transient failures occur, and increases the availability of workloads.
+
+    FAR is one of the remediator operators by [Medik8s](https://www.medik8s.io/remediation/remediation/),
+    such as [Self Node Remediation](https://github.com/medik8s/self-node-remediation) and [Machine Deletion Remediation](https://github.com/medik8s/machine-deletion-remediation),
+    that were designed to run with the Node HealthCheck Operator [(NHC)](https://github.com/medik8s/node-healthcheck-operator) which detects an unhealthy node and creates remediation CR.
+    It is recommended to use FAR with NHC for an easier and smoother experience by fully automate the remediation process, but it can be used as a standalone remediator for the more experienced user.
+
+    FAR is recommended when a node becomes unhealthy, and we want to completely fence/isolate the node from a cluster, since we can not “trust” the unhealthy node,
+    to prevent it from accessing the shared resources like [RWO volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).
+    Moreover, FAR is *robust* as it can remediate an unhealthy node using a traditional API (e.g., IPMI) while still keeping control plane connectivity,
```

Comment:
Not sure what you mean by "traditional API" :thinking_face:

Reply:
As regular/known API. But maybe the `traditional` isn't useful here.